user_tasks/conf.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+"""
+Custom Django settings for django-user-tasks.
+"""
+
+from __future__ import absolute_import, unicode_literals
+
+from datetime import timedelta
+
+from django.conf import settings as django_settings
+from django.core.files.storage import get_storage_class
+
+from user_tasks import filters
+
+
+class LazySettings(object):
```

Comment:
This is a naming convention that I've seen (and used) in several other open source projects, which probably all copied it from each other inspired by the fact that Django's settings object is an instance of a `LazySettings` class.  Really it's just a reminder that the properties won't be evaluated until accessed.